rollup-examples.config.js

```diff
@@ -0,0 +1,68 @@
+var path = require( 'path' );
+var fs = require( 'fs' );
+
+// Creates a rollup config object for the given file to
+// be converted to umd
+function createOutput( file ) {
+
+	var inputPath = path.resolve( file );
+	var outputPath = inputPath.replace( /[\\\/]examples[\\\/]jsm[\\\/]/, '/examples/js/' );
+
+	// Every import is marked as external so the output is 1-to-1. We
+	// assume that that global object should be the THREE object so we
+	// replicate the existing behavior.
+	return {
+
+		input: inputPath,
+		treeshake: false,
+		external: p => p !== inputPath,
+
+		output: {
+
+			format: 'umd',
+			name: 'THREE',
+			file: outputPath,
+
+			globals: () => 'THREE',
+			extend: true,
+
+			banner:
+				'/**\n' +
+				` * Generated from '${ path.relative( '.', inputPath.replace( /\\/, '/' ) ) }'\n` +
+				' **/\n'
```

Comment:
nit: typically the header would end in `*/` rather than `**/` I think

Reply:
Fixed this, too! Thanks